### PR TITLE
Add a paragraph about render methods naming

### PR DESCRIPTION
**Pull request by DmitryKorlas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -573,3 +573,48 @@ export class MyAwesomeComponent extends React.PureComponent {
     }
 }
 ```
+
+### renderXXX
+
+For better navigation, use `render` prefix for any method who returns JSX or component.
```
Comment (ledamint) [inline README.md]: `who` -> `which`

File: README.md [hunk 2]
```diff
@@ -573,3 +573,48 @@ export class MyAwesomeComponent extends React.PureComponent {
     }
 }
 ```
+
+### renderXXX
+
+For better navigation, use `render` prefix for any method who returns JSX or component.  
+Of course it is possible to use another prefixes like `get` or even `display`, but in terms of React we recommend to use `renderXXX`.
```
Comment (ledamint) [inline README.md]: I suggest to make using `render` strict without alternative

File: README.md [hunk 3]
```diff
@@ -573,3 +573,48 @@ export class MyAwesomeComponent extends React.PureComponent {
...
+        
+        return (
+            <Block>
+                <h3>{title}</h3>
+                <p>{description}</p>
+                <Image src={url} />
+            </Block>
+        );
+    }
+
```
Comment (ledamint) [inline README.md]: Let's add BAD examples

File: README.md (hunk 1 above)
Comment (DmitryKorlas) [reply README.md]: thx)

File: README.md (hunk 2 above)
Comment (DmitryKorlas) [reply README.md]: I too. Do you suggest to remove this phrase?

File: README.md (hunk 3 above)
Comment (DmitryKorlas) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (DmitryKorlas) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (DmitryKorlas) [reply README.md]: done

**Pull request merged**